src/components/Icon.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable no-unused-vars */
+import React from 'react'
+import styled from 'styled-components'
```

Comment:
Suggestion: If you add macro i.e:
import styled from 'styled-components/macro'; 
YOu will see the different names of your components instead of figures.

Reply:
Oh sweet, thank you so much! The random class names were making things difficult so this is wonderful 😀